pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -97,6 +97,9 @@ type Manager interface {
 	// GetAllCPUs returns all the CPUs known by cpumanager, as reported by the
 	// hardware discovery. Maps to the CPU capacity.
 	GetAllCPUs() cpuset.CPUSet
+
+	// Check whether any container within a pod uses exclusive cpus
+	PodContainsPinnedCpus(pod *v1.Pod) bool
```

Comment:
Is this needed now? I think we want to use the equivalent `HasExclusiveCPUs` interface, if so we can drop this earlier suggestion